<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameDialog</class>
 <widget class="QDialog" name="RenameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>120</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rename Project</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: rgb(43, 45, 48);
    color: white;
}

QLabel {
    color: white;
    font-family: &quot;Pixel Mixed&quot;;
    font-size: 12px;
}

QLineEdit {
    background-color: rgb(43, 45, 48);
    color: white;
    padding: 5px;
    border: 1px solid rgb(85, 90, 102);
    border-radius: 4px;
    font-family: &quot;Pixel Mixed&quot;;
    font-size: 12px;
}

QPushButton {
    background-color: #333842;
    border: 1px solid #555A66;
    border-radius: 6px;
    color: white;
    padding: 6px 6px;
    font-family: &quot;Pixel Mixed&quot;;
    font-size: 12px;
}

QPushButton:hover {
    background-color: #4B5362;
    color: white;
}

QPushButton:pressed {
    background-color: #2B2F38;
    color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>New Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
